src/components/DocumentationTopic/ViewMore.vue

```diff
@@ -0,0 +1,42 @@
+<!--
+  This source file is part of the Swift.org open source project
+
+  Copyright (c) 2023 Apple Inc. and the Swift project authors
+  Licensed under Apache License v2.0 with Runtime Library Exception
+
+  See https://swift.org/LICENSE.txt for license information
+  See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+-->
+
+<template>
+  <div class="view-more-link">
+    <router-link
+      :to="url"
+      class="base-link"
+    >
+      View more
+    </router-link>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'ViewMore',
+  props: {
+    url: {
+      type: String,
+      required: true,
+    },
+  },
+};
+</script>
+
+<style scoped lang="scss">
+@import 'docc-render/styles/_core.scss';
+
+.base-link {
+  float: right;
```

Comment:
Wow, 🚀  time travel. Any way we can use something ... less ancient? 🤣

Reply:
Me trying to remember to use flex box for most things these days:
![me](https://media.giphy.com/media/v1.Y2lkPTc5MGI3NjExYWIyMDE1M2RlMDI2ZWNiNDk1ZmYyN2Q4ZmMyNjJlZDJiZTdjMzgzZCZjdD1n/6uIqPGAUYFztGBjxEi/giphy.gif)

Reply:
Pulled it back to modern time :P